### PR TITLE
Global style

### PR DESCRIPTION
**Pull request by jeremyjorand**

### CODE REVIEW

**Review by mastemm — DISMISSED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
-      <HomePage />
+    <SGlobal className="App">
       <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/marvel" element={<Marvel />} />
+        <Route path="/starwars" element={<Starwars />} />
+        <Route path="/manga" element={<Manga />} />
+        <Route path="/dccomics" element={<Dccomics />} />
+        <Route path="/autresheros" element={<Autresheros />} />
+
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: ???

File: frontend/src/components/HomePage/HomePageHeader/index.jsx [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+            <li className="manga"> </li>
+          </Link>
+          <Link to="/dccomics">
+            <li className="dccomics"> </li>
+          </Link>
+          <Link to="/autresheros">
+            <li className="autresheros"> </li>
+          </Link>
+        </ul>
+        {/*  */}
```
Comment (LuigiMenez) [inline frontend/src/components/HomePage/HomePageHeader/index.jsx]: No

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Remove Dead code line

**Review by mastemm — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**